styles.css

```diff
@@ -34,6 +34,12 @@ body {
 .branding {
   color: var(--blue);
 }
+.logo-text {
+  margin-left: 4rem;
+}
+.nav-right {
+  margin-right: 0;
```

Comment:
Are you resetting margin here, or setting 0 here. If setting here then I think 0 is by default naa.

Reply:
Actually,I am resetting the margin here just for the landing page.

Reply:
Got it. Everything seems good, Go ahead.